Makefile

```diff
@@ -281,19 +281,14 @@ run-android: export ORG_GRADLE_PROJECT_hermesEnabled := false
 # INFO: If it's empty (no devices attached, parsing issues, script error) - for Nix it's the same as not set.
 run-android: export ANDROID_ABI_INCLUDE ?= $(shell ./scripts/adb_devices_abis.sh)
```

Comment:
We can Probably move this into the script to avoid spawning an extra Nix shell.

Reply:
addressed here : https://github.com/status-im/status-mobile/pull/18784/commits/d6a19d75da1010ab9e33dd395ba4c6e45e7456e6